examples/carla/docker-compose.yml

```diff
@@ -2,9 +2,10 @@ version: "3.3"
 
 services:
   server:
-    build:
-      context: .
-      dockerfile: Dockerfile
+    image: "carlasim/carla"
```

Comment:
How about using `carlasim/carla:0.9.13` here too?

Reply:
Sure, i was trying to save you guys some time to download another image